packages/tasit-sdk/src/testHelpers/helpers.js

```diff
@@ -1,23 +1,42 @@
+import { ethers } from "ethers";
 import { Account, Action } from "../TasitSdk";
-const { Contract, NFT, ERC20, ERC721, Marketplace } = Action;
+const { Contract, ERC20, ERC721, Marketplace } = Action;
 const { Mana } = ERC20;
 const { Estate, Land } = ERC721;
 const { Decentraland } = Marketplace;
 import { createFromPrivateKey } from "tasit-account/dist/testHelpers/helpers";
 
+// The goal of this integration test suite is to use only exposed classes
+// from TasitSdk. ProviderFactory is used here as an exception
+// as the clearest way to get a provider
+// in this test suite. Eventually, maybe ProviderFactory may move to
```

Comment:
Not yet but it's a good point to talk about. A think that now is a good moment to merge test helpers in one place because there is some duplicated code. Which project do you think is a good place to put that?

Reply:
Note: I think that some helpers functions could be utils functions exposed from the SDK (I'll comment these from now).

Reply:
I agree that now is a good moment for that. OpenZeppelin has a special repo for their test helpers, so maybe we could even make them their own child package to follow their example but with more of a monorepo style.

If we do want to keep them in an existing package, we can use this question to frame it: Do the test helpers end up being used more in unit tests or integration tests? Or a little of both?

If unit, maybe action. If integration, maybe tasit SDK. If mixture, probably a new child package.

And I agree that some helpers functions could be utils functions exposed from the SDK. And it would be great if put comments near them about that like you mentioned.

Reply:
https://github.com/tasitlabs/TasitSDK/issues/212

Reply:
Thanks! Resolving